.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Set update schedule for GitHub Actions
```

Comment:
I am not sure if `directory: /` would also work for https://github.com/grafana/plugin-ci-workflows/blob/814f763609866edfb5df79d7e9a1eed7609bb210/actions/plugins/setup/action.yml#L19

but I guess we just have to try it out and see